Tests/SentryTests/Integrations/Performance/Network/SentryNetworkTrackerTests.swift

```diff
@@ -806,19 +806,25 @@ class SentryNetworkTrackerTests: XCTestCase {
         let transaction = startTransaction()
 
         let queue = DispatchQueue(label: "SentryNetworkTrackerTests", qos: .userInteractive, attributes: [.concurrent, .initiallyInactive])
-        let group = DispatchGroup()
 
-        for _ in 0...100_000 {
-            group.enter()
+        let loopCount = 1_000
```

Comment:
`m`: Why changing the loop count too?

Reply:
100_000 is a bit much for CI and not really required. If you really want to test thread-safety you anyways increase the loop count to even higher value and you must run the test repeatedly.

Sorry, I should have added a comment for this in the PR.